services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/Page.java

```diff
@@ -109,4 +141,57 @@ public String toString() {
                        .add("items", items)
                        .build();
     }
+
+    public Integer count() {
```

Comment:
Missing Javadoc, same for other methods below.

Reply:
I went through the code to check for javadocs, missed these. Well spotted.